src/README.md

```diff
@@ -0,0 +1,27 @@
+## Welcome to the Babel Codebase
+
+Babel is broken down into three parts:
+
+- Parsing
+- Transformation
+- Generation
+
+**Parsing** is the process of turning a piece of code into an AST (Abstract
+Syntax Tree) which is a tree-like object of nodes that _describes_ the code.
```

Comment:
I think it will be helpful if there will be a link to [ESTree spec](https://github.com/estree/estree)